src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -11,12 +11,14 @@
   "frameworks": {
     "dnxcore50": {
       "dependencies": {
+        "Microsoft.DotNet.xunit.performance.analysis.cli": "1.0.0-alpha-build0032",
```

Comment:
This package depends on 
```
<dependency id="Microsoft.NETCore.App" version="[1.0.0-rc2-3002520, )" />
```
I think you should trim this down to just the packages you actually depend on.  @weshaggard what do you think?